tf2onnx/tfonnx.py

```diff
@@ -471,6 +471,7 @@ def rewrite_incomplete_type_support_rs6(g, ops):
         "Split",
         "Tile",
         "Transpose",
+        "Gemm",
```

Comment:
do we need it now?

Reply:
I have check the problem and make sure we do not need "Gemm" in this function.